MSAL/test/unit/native_auth/mock/MSALNativeAuthResetPasswordControllerMock.swift

```diff
@@ -28,24 +28,24 @@ import XCTest
 
 class MSALNativeAuthResetPasswordControllerMock: MSALNativeAuthResetPasswordControlling {
 
-    var resetPasswordResult: ResetPasswordStartResult!
-    var resendCodeResult: ResetPasswordResendCodeResult!
-    var submitCodeResult: ResetPasswordVerifyCodeResult!
-    var submitPasswordResult: ResetPasswordRequiredResult!
+    var resetPasswordResult: ResetPasswordStartControllerResponse!
```

Comment:
Should we rename all of these to Response as they are now responses?

Reply:
Yes, nice catch. Done.